tests/gpflow/likelihoods/test_heteroskedastic_constant_variance.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2017-2020 the GPflow authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+import gpflow
+from gpflow.likelihoods.heteroskedastic import HeteroskedasticTFPDistribution
+
+tf.random.set_seed(99012)
+
+
+class Data:
+    g_var = 0.345
+    # toData.Y data: shape [N, 1]
+    Y = np.c_[0.5, 0.4, 1.0].T
+    N, _ = Y.shape
+    # single "GP" (for the mean):
+    f_mean = np.c_[0.4, 0.7, 0.9].T
+    f_var = np.c_[0.7, 0.2, 1.3].T
+    equivalent_f2 = np.log(np.sqrt(0.345))
```

Comment:
```suggestion
    equivalent_f2 = np.log(np.sqrt(g_var))
```
We can re-use `g_var` which makes clearer what's happening (and more robust to changing the value above...)

Reply:
Whoops, that was why I pulled out the g_var constant, and then forgot to use it. Thanks!